sqlscripts/replication/0001-jaeger-index-local.sql

```diff
@@ -5,8 +5,13 @@ CREATE TABLE IF NOT EXISTS %s ON CLUSTER '{cluster}'
     service    LowCardinality(String) CODEC (ZSTD(1)),
     operation  LowCardinality(String) CODEC (ZSTD(1)),
     durationUs UInt64 CODEC (ZSTD(1)),
-    tags       Array(String) CODEC (ZSTD(1)),
-    INDEX idx_tags tags TYPE bloom_filter(0.01) GRANULARITY 64,
+    spankind LowCardinality(String),
```

Comment:
why is the span.kind treated differently than other tags? Search by any tag key and value should work as well. Could we use that instead?